engine/src/flutter/impeller/renderer/capabilities.h

```diff
@@ -126,6 +126,11 @@ class Capabilities {
   /// @brief The minimum alignment of uniform value offsets in bytes.
   virtual size_t GetMinimumUniformAlignment() const = 0;
 
+  /// @brief The minimum alignment of storage buffer value offsets in bytes.
+  virtual size_t GetMinimumStorageAlignment() const {
```

Comment:
`GetMinimumStorageBuffer` alignment? Or even `GetMinimumSSBO` alignment? I wasn't sure what storage was without a lot of context.